src/gui/entry/EditEntryWidgetAdvanced.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>400</width>
-    <height>366</height>
+    <width>448</width>
+    <height>370</height>
```

Comment:
You should completely remove widget sizes here (click the little reset button in QtDesigner's property view).